nhais/outbound/schema/schema_validation_exception.py

```diff
@@ -0,0 +1,6 @@
+
+class SchemaValidationException(Exception):
```

Comment:
Since we're not using JSONSchema anymore it might make sense to rename to 'RequestValidationException' or similar?

Reply:
Renamed